packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2250,14 +2257,22 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
      */
     public async submitSummary(options: ISubmitSummaryOptions): Promise<SubmitSummaryResult> {
         const { fullTree, refreshLatestAck, summaryLogger } = options;
+        const summaryNumberLogger = ChildLogger.create(
+            summaryLogger,
+            undefined,
+            {
+                all: { summaryNumber: this.nextSummaryNumber },
+            },
+        );
```

Comment:
Why are we creating a new child logger?

Reply:
To add `summaryNumber` to all events generated during summarize.